docs/devel/development.md

```diff
@@ -22,7 +22,9 @@ $ git clone https://github.com/GoogleCloudPlatform/kubernetes.git
 
 The commands above will not work if there are more than one directory in ``$GOPATH``.
 
-(Obviously, clone your own fork of Kubernetes if you plan to do development.)
+If you plan to do development, read about the
+[Kubernetes Github Flow](https://docs.google.com/a/google.com/presentation/d/1WDGN_ggq1Ae3eeQmbSCMyUG1UhhRH6UZTy0pePq09Xo/pub?start=false&loop=false&delayms=3000),
```

Comment:
This presentation is not visible outside google.com - it's a restriction of our domain.  You can either create it from a private gmail account or render to a less beautiful format like multi-page PDF or maybe animated gif?